CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.2.4
+-----
+**Date:** 2017-02-..
+
+Charge of bound electrons, openPMD Axis Range
```

Comment:
this line is the title of the release will proper caps of all words but names and prepositions